### PR TITLE
Create consortium.schema.tpl.json

### PR DESCRIPTION
**Pull request by lzehl**
related to #360 (second try to do this PR :see_no_evil: )

### CODE REVIEW

**Comment by jcolomb** [comment]
is this supposed to be an author of the dataset?

in jatsxml, collaboration group can have an affiliation and can have no member described.

**Comment by lzehl** [comment]
@jcolomb yes. It came in as a request. So you would add an affiliation (optional) and keep members as optional? 
Would that not be then an Organization (all members with same affiliation) compared to a Consortium (members with different affiliations)? Or differently asked what is the affiliation of a Consortium with members with different affiliations? And a Consortium should always have members no? They might not be known though (why keeping it option makes sense).

I'm asking because we can currently have as "contributors" (authors, developers) Organizations and Persons. Consortium would be the third new option. Not sure to which the JATS schema(s) would relate best. Could you clarify?

**Comment by jcolomb** [comment]
jats only has one entry for organisation and consortiums (called "collaborative group" I think), therefore they pack probably both types of entities under the same schema.

the issue with members is that it may change over time, the groups were not well described yet, especially in terms of what happens when the member list change.

**Comment by lzehl** [comment]
@jcolomb the changing list of members of a consortium does not matter considering authorships I think. The legal person is here the consortium itself (independent of it's members). The members are just an additional information which indeed can change over time. 

@apdavison what are your thoughts here? do you have an idea if we could track such changes though over time for a consortium?

**Comment by apdavison** [comment]
I would use the affiliation mechanism

**Comment by lzehl** [comment]
@apdavison meaning you would not add the properties 'members' to a Consortium schema but rather point from Person to a Consortium via Affiliation ?

**Comment by apdavison** [comment]
from Person and from Organization, yes. Affiliation has start date and end date, so allows membership to evolve over time.

**Comment by lzehl** [comment]
@apdavison I've deleted the 'member' property from Consortium now, changed the property 'organization' in Affiliation schema to 'memberOf' and added Consortium there as linked type besides Organization.

What should we do with Organization. This schema currently does not have a link to Affiliation (which I think also does not make sense). Should I add there directly the property 'memberOf' with linked type to Consortium?

**Comment by apdavison** [comment]
I think that adding Affiliation to Organization makes perfect sense, e.g. FZJ (organization) is affiliated to HBP (consortium).

**Comment by lzehl** [comment]
@apdavison I don't think this is true... but maybe I have a different understanding of affiliation...

**Comment by lzehl** [comment]
@apdavison I've added now affiliation to organizations as well

File: schemas/actors/affiliation.schema.tpl.json [hunk 1]
```diff
@@ -18,10 +18,11 @@
         "type": "string",
         "_instruction": "Enter the end date of this affiliation. Leave blank if the affiliation is still current."
      },
-     "organization": {
-       "_instruction": "Add organization to which a person is or was affiliated.",
+     "memberOf": {
+       "_instruction": "Add the organization or consortium to which a person is or was affiliated.",
```
Comment (apdavison) [inline schemas/actors/affiliation.schema.tpl.json]: or the consortium to which an organization is affiliated

Comment (lzehl) [reply schemas/actors/affiliation.schema.tpl.json]: done

**Review by apdavison — APPROVED** [review]

**Pull request merged**